pkg/runtime/runtime.go

```diff
@@ -1378,6 +1378,10 @@ func (a *DaprRuntime) preprocessOneComponent(comp *components_v1alpha1.Component
 // Stop allows for a graceful shutdown of all runtime internal operations or components
 func (a *DaprRuntime) Stop() {
 	log.Info("stop command issued. Shutting down all operations")
+
```

Comment:
I realized that dapr runtime doesn't do anything for the shutdown process. we have to do some graceful shutdown operaitons. such as shutting down internal grpc server and http/grpc api servers.

Reply:
Please create issue to track it

Reply:
https://github.com/dapr/dapr/issues/2391 here.